types/factory-girl/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-duplicate-imports": false
```

Comment:
I need this suppression for testing `import *` and `import {...}` in the same file.

Reply:
Makes sense, but can you use a comment to disable it only where needed?

Reply:
@andrewbranch Thanks for your comment. I fixed it in e8881a9.